### ISSUE TITLE
Rollback not working properly

### ISSUE THREAD

**Issue by krzkrzkrz** [MSch/devise-twitter#11]
If one decides to rollback. An error appears:

rake aborted!
An error has occurred, this and all later migrations canceled:

PGError: ERROR:  column "string" of relation "users" does not exist
: ALTER TABLE "users" DROP "string"

---

The problem is at:

class AddDeviseTwitterFieldsToUsers < ActiveRecord::Migration

 ......

```
change_table(:users) do |t|
  t.remove :twitter_handle, :string
  t.remove :twitter_oauth_token, :string
  t.remove :twitter_oauth_secret, :string
end
```

  end

it should be:

```
change_table(:users) do |t|
  t.remove :twitter_handle
  t.remove :twitter_oauth_token
  t.remove :twitter_oauth_secret
end
```

The above works if changed. Thought you might want to look at this...

**Comment by MSch** [comment]
Fix migration (closed by 8a733e03ecac554ea3afca7775a8b12abb37a38f, thanks krzkrzkrz!)

**Comment by MSch** [comment]
Fixed, thanks!
